pxd/zpool_status.pxi.in

```diff
@@ -0,0 +1,43 @@
+#
+# The zpool_status_t enum has a lot of different variations.
+# We resort to using autoconf for the metaprogramming, because
+# Cython lacks the expressivity to conditionally define enumerators.
+#
+
+cdef extern from "libzfs.h" nogil:
+    ctypedef enum zpool_status_t:
+        ZPOOL_STATUS_CORRUPT_CACHE
+        ZPOOL_STATUS_MISSING_DEV_R
+        ZPOOL_STATUS_MISSING_DEV_NR
+        ZPOOL_STATUS_CORRUPT_LABEL_R
+        ZPOOL_STATUS_CORRUPT_LABEL_NR
+        ZPOOL_STATUS_BAD_GUID_SUM
+        ZPOOL_STATUS_CORRUPT_POOL
+        ZPOOL_STATUS_CORRUPT_DATA
+        ZPOOL_STATUS_FAILING_DEV
+        ZPOOL_STATUS_VERSION_NEWER
+        ZPOOL_STATUS_HOSTID_MISMATCH
+        ZPOOL_STATUS_HOSTID_ACTIVE
+        ZPOOL_STATUS_HOSTID_REQUIRED
+        ZPOOL_STATUS_IO_FAILURE_WAIT
+        ZPOOL_STATUS_IO_FAILURE_CONTINUE
+        ZPOOL_STATUS_IO_FAILURE_MMP
+        ZPOOL_STATUS_BAD_LOG
+        @ZPOOL_STATUS_ERRATA@
+        ZPOOL_STATUS_UNSUP_FEAT_READ
+        ZPOOL_STATUS_UNSUP_FEAT_WRITE
+        ZPOOL_STATUS_FAULTED_DEV_R
+        ZPOOL_STATUS_FAULTED_DEV_NR
+        ZPOOL_STATUS_VERSION_OLDER
+        ZPOOL_STATUS_FEAT_DISABLED
+        ZPOOL_STATUS_RESILVERING
+        ZPOOL_STATUS_OFFLINE_DEV
+        ZPOOL_STATUS_REMOVED_DEV
+        @ZPOOL_STATUS_REBUILDING@
+        @ZPOOL_STATUS_REBUILD_SCRUB@
+        ZPOOL_STATUS_NON_NATIVE_ASHIFT
+        @ZPOOL_STATUS_COMPATIBILITY_ERR@
+        @ZPOOL_STATUS_INCOMPATIBLE_FEAT@
+        ZPOOL_STATUS_OK
+
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
```

Comment:
We could probably remove this ?

Reply:
Could, but it makes editing the code easier and doesn't hurt to have it. The whitespace matters in these files (it's effectively Python) but vim doesn't recognize ".pxi.in" so defaults to indenting with tabs instead of spaces.